version.py

```diff
@@ -78,4 +99,10 @@ def get_version(prefix='', cwd='.'):
 
 
 if __name__ == "__main__":
-    print(get_version(prefix='v'))
+    if '--no-update-version-file' in sys.argv and len(sys.argv) == 2:
+        update_version_file = False
+    elif len(sys.argv) == 1:
+        update_version_file = True
+    else:
+        raise ValueError('Invalid arguments')
```

Comment:
We should probably use [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) instead of mocking our own solution from scratch.

Reply:
@jiivan Done, if it is OK, just merge it please.